readme.md

```diff
@@ -35,6 +35,7 @@ election
 fbi
 federal agencies
 federal agency
+feminis
```

Comment:
intended to match `feminism` and `feminist`
I'll update with two separate words